templates/vsan_claim.py

```diff
@@ -73,7 +73,7 @@ def CollectMultiple(content, objects, parameters, handleNotFound=True):
 
 
 si = connect.SmartConnectNoSSL(
-    host=vcenter_fqdn, user=vcenter_user, pwd=vcenter_pass, port=443
```

Comment:
So one thing here... It looks like you might be mixing up the vCenter Pass and the SSO Pass

There is a `root` password to the vCenter Virtual Machine that is different from the vCenter Single Sign On (SSO) password.

Reply:
Oh.. on second inspection this looks correct.

Reply:
The `vcenter_pass` variable was confusing because other scripts used `vcenter_password`.  This variable was the `sso_password` so the name now matches.